apps/workflow-helper.sh

```diff
@@ -11,6 +11,10 @@ registry_password=$(sed -nr 's|.*\bregistry_password=(\S+).*|\1|p' /proc/cmdline
 tinkerbell=$(sed -nr 's|.*\btinkerbell=(\S+).*|\1|p' /proc/cmdline)
 worker_id=$(sed -nr 's|.*\bworker_id=(\S+).*|\1|p' /proc/cmdline)
 id=$(curl --connect-timeout 60 "$tinkerbell:50061/metadata" | jq -r .id)
+log_driver=$(sed -nr 's|.*\blog_driver=(\S+).*|\1|p' /proc/cmdline)
+log_opt_tag=$(sed -nr 's|.*\blog_opt_tag=(\S+).*|\1|p' /proc/cmdline)
+log_opt_server_address=$(sed -nr 's|.*\blog_opt_server_address=(\S+).*|\1|p' /proc/cmdline)
+log_opt_server_address_type=$(sed -nr 's|.*\blog_opt_server_address_type=(\S+).*|\1|p' /proc/cmdline)
```

Comment:
I thought we came to the agreement that we would not add the `address_type` arg and instead do
```
--log-opt "$log_driver-address=$log_opt_server_address
```

Also I thought we were going to go with
```
LOG_DRIVER
LOG_TAG
LOG_SERVER_ADDRESS
LOG_OPT_(all the others
```
I'm ok with doing `LOG_DRIVER` and `LOG_OPT` for all the options just wanted to ask.

Reply:
I have updated the `address-type`. 

For options, I think we concluded to have `OPT` for all the `log-options`.